### DISCUSSION TITLE
Almost perfect prediction

### DISCUSSION THREAD

**Discussion by fersebas** [PaddlePaddle/PaddleSpatial#240] [BaiduKDDCup2022]
What is hapenning??? WTF, maybe some leak test data?

<img width="894" alt="image" src="https://user-images.githubusercontent.com/49498032/177595436-6bfb2d93-f2e7-44ab-a304-da0f187cda1b.png">

**Comment by shaido987** [comment]
This submission have now disappeared from the leaderboard, so for sure there was some type of issue with it.

**Reply by fersebas** [reply]
Ah ok, make sense ;)
